master/getting-started/kubernetes/reqs/system.md

```diff
@@ -0,0 +1,39 @@
+---
+title: System requirements
+canonical_url: 'https://docs.projectcalico.org/v3.0/getting-started/kubernetes/reqs/system'
+---
+
+{% include {{page.version}}/reqs-sys.md orch="Kubernetes" %}
+
+## Kubernetes
+
+#### Supported versions
+
+{{site.prodname}} {{page.version}} requires Kubernetes 1.8 or later.
+
+#### CNI plug-in enabled
+
+Enable CNI by passing `--network-plugin=cni` to kubelet. On kubeadm, this is
```

Comment:
I don't think we need to or should call out specific installers (e.g. kubeadm) here.

Reply:
Might be nice to have a bit of context - "Calico is installed as a CNI plugin, and so the kubelet must be configured to use CNI networking by passing the `--network-plugin=cni` argument."

Not really bothered either way, so up to you.